concepts/array/introduction.md

```diff
@@ -134,7 +134,7 @@ If the array is empty an `IndexError` will be raised.
 
 ```crystal
 numbers = [1, 2, 3]
-[1, 2, 3].pop # => 3
+numbers.pop # => 3
 numbers       # => [1, 2]
```

Comment:
This line is different; the document-generator CLI makes these types of changes really easy. Then you only have to modify the about file and then the script will do the rest.

Reply:
Is it the `configlet` CLI? I tried to run `configlet sync --docs --update` but it didn't update the doc, I did it manually.

Reply:
Oh never mind, I thought it was some unified exercism tooling, but I found the `document-generator/README.md`.
I compiled it and it worked. I'll know how to use it next time.